models/app-models.js

```diff
@@ -44,4 +42,37 @@ const fetchReviewsbyId = (req) => {
     });
 };
 
-module.exports = { fetchCategories, fetchReviews, fetchReviewsbyId };
+const fetchReviewIdComments = (req) => {
+  const { params } = req;
```

Comment:
should the model have access to the request? I would argue that that is the job of the controller